index.bs

```diff
@@ -1472,12 +1472,19 @@ interface WebTransportError : DOMException {
   constructor(optional WebTransportErrorInit init = {});
 
   readonly attribute octet? applicationProtocolCode;
+  readonly attribute WebTransportErrorSource source;
 };
 
 dictionary WebTransportErrorInit {
   octet applicationProtocolCode;
   DOMString message;
 };
+
+enum WebTransportErrorSource {
+  "stream",
+  "session",
+  "connection",
```

Comment:
I think these values need explanations. What's the difference between `"session"` vs. `"connection"`, and how might an application react differently?